src/libraries/System.Configuration.ConfigurationManager/src/System.Configuration.ConfigurationManager.csproj

```diff
@@ -245,11 +246,28 @@
     <Compile Include="System\Configuration\SettingsSerializeAs.cs" />
     <Compile Include="System\Configuration\NoSettingsVersionUpgradeAttribute.cs" />
     <Compile Include="System\UriIdnScope.cs" />
+    <Compile Include="$(CommonPath)Interop\Windows\Interop.Errors.cs">
```

Comment:
Will using the Windows errors for both Win and Unix work fine?

Reply:
Never mind I see that this is used on Windows only but with runtime checks.

Reply:
Since this library only builds for `netstandard2.0`, and doesn't have specific OS builds, I opted for using `RuntimeInformation.IsOSPlatform` to check before calling into Windows APIs.

Reply:
Yeah, I think that's fine.